src/main/java/Classes/Auto.java

```diff
@@ -65,8 +67,48 @@ public String toString() {
                 ", dateInspection=" + dateInspection +
                 '}';
     }
-}
 
+    public void driveAbility() {
+        if (tankFullness > 1) {
+            System.out.println("Lets go \uD83D\uDCA8\uD83D\uDCA8\uD83D\uDCA8");
+        } else {
+            System.out.println("Better to refuel this car!");
+        }
+    }
+
+    public void exploitionAbility() {
+        Period period = Period.between(dateInspection, currentDate);
+
+        if (period.getYears() > 2 || (period.getYears() == 2 && (period.getMonths() > 0 || period.getDays() > 0))) {
+            System.out.println("The technical inspection has expired, you need to go through it again");
+        } else if (period.getYears() < 2 || (period.getYears() == 2 && period.getMonths() == 0 && period.getDays() == 0)) {
+            System.out.println("The technical inspection is still valid");
+        } else {
+            System.out.println("==");
+        }
+
+        LocalDate endDate = dateInspection.plusYears(2);
+        Period remainingPeriod = Period.between(currentDate, endDate);
+
+        if (remainingPeriod.isNegative()){
+        } else {
+            System.out.println("Time remaining until two-year period: " +
+            remainingPeriod.getYears() + " years, " +
+            remainingPeriod.getMonths() + " months, and " +
+            remainingPeriod.getDays() + " days.");
+        }
+    }
+
+    public void howFarTravel() {
+        if (tankFullness > 1) {
+            double distance = (double) tankFullness / fuelConsumption * 100;
```

Comment:
`fuelConsumption` is already double and the whole expression will be casted to double. 
Therefore no need to add explicit type casting.